docs/sources/user_guide/regressor/LinearRegression.ipynb

```diff
@@ -129,7 +129,7 @@
     "\n",
     "$$\\mathbf{w} = \\mathbf{X}^+  \\mathbf{y}.$$\n",
     "\n",
-    "Please note that this OLS method is computationally most inefficient. However, it is a useful approach when the direct method (normal equations) or QR factorization cannot be applied or the normal equations (via $\\mathbf{X}^T \\mathbf{X}$) are ill-conditioned.\n",
+    "Please note that this OLS method is computationally most inefficient. However, it is a useful approach when the direct method (normal equations) or QR factorization cannot be applied or the normal equations (via $\\mathbf{X}^T \\mathbf{X}$) are ill-conditioned[3].\n",
```

Comment:
I suggest adding a whitespace between "ill-conditioned" and "[3]" for consistency

Reply:
I did change it actually, in this commit:
https://github.com/rasbt/mlxtend/pull/575/commits/bc93516793fd4d4e5b2371dc0205eb3f62ccf6da